### PR TITLE
Release `v0.10.0.post1`

### PR DESCRIPTION
**Pull request by angrybayblade**
## Proposed changes

- Adds ledger connection overrides in the agent and the service configurations
- Bumps `open-autonomy` framework
- Sets the `p2p_libp2p_client` as none abstract to enable `ACN`

## Fixes

If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce? (A **breaking change** is a fix or feature that would cause existing functionality and APIs to not work as expected.)
_Put an `x` in the box that applies_

- [ ] Non-breaking fix (non-breaking change which fixes an issue)
- [ ] Breaking fix (breaking change which fixes an issue)
- [ ] Non-breaking feature (non-breaking change which adds functionality)
- [ ] Breaking feature (breaking change which adds functionality)
- [ ] Refactor (non-breaking change which changes implementation)
- [ ] Messy (mixture of the above - requires explanation!)

## Checklist

_Put an `x` in the boxes that apply._

- [ ] I have read the [CONTRIBUTING](../blob/main/CONTRIBUTING.md) doc
- [ ] I am making a pull request against the `main` branch (left side). Also you should start your branch off our `main`.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have locally run services that could be impacted and they do not present failures derived from my changes

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: packages/valory/services/apy_estimation_demo/service.yaml [hunk 1]
```diff
@@ -105,9 +103,40 @@ type: skill
...
-        share_tm_config_on_startup: ${USE_ACN:bool:false}
+        service_registry_address: ${SERVICE_REGISTRY_ADDRESS:str:null}
+        share_tm_config_on_startup: ${SHARE_TM_CONFIG_ON_STARTUP:bool:false}
         on_chain_service_id: ${ON_CHAIN_SERVICE_ID:int:null}
         tendermint_p2p_url: ${TM_P2P_ENDPOINT_NODE_3:str:node3:26686}
     server_api:
       args: *id002
+---
+public_id: valory/ledger:0.19.0
+type: connection
```
Comment (angrybayblade) [inline packages/valory/services/apy_estimation_demo/service.yaml:114]: Overrides for ledger connection were missing

File: packages/valory/services/apy_estimation/service.yaml [hunk 2]
```diff
@@ -105,9 +103,40 @@ type: skill
...
-        share_tm_config_on_startup: ${USE_ACN:bool:false}
+        service_registry_address: ${SERVICE_REGISTRY_ADDRESS:str:null}
+        share_tm_config_on_startup: ${SHARE_TM_CONFIG_ON_STARTUP:bool:false}
         on_chain_service_id: ${ON_CHAIN_SERVICE_ID:int:null}
         tendermint_p2p_url: ${TM_P2P_ENDPOINT_NODE_3:str:node3:26686}
     server_api:
       args: *id002
+---
+public_id: valory/ledger:0.19.0
+type: connection
```
Comment (angrybayblade) [inline packages/valory/services/apy_estimation/service.yaml:114]: Same

File: packages/valory/agents/apy_estimation/aea-config.yaml [hunk 3]
```diff
@@ -137,4 +137,12 @@ cert_requests:
   not_before: '2023-01-01'
   public_key: ${str:02d3a830c9d6ea1ae91936951430dee11f4662f33118b02190693be835359a9d77}
   save_path: .certs/acn_cosmos_9005.txt
-is_abstract: true
+---
+public_id: valory/ledger:0.19.0
+type: connection
```
Comment (angrybayblade) [inline packages/valory/agents/apy_estimation/aea-config.yaml:142]: same

File: packages/valory/services/apy_estimation_demo/service.yaml (hunk 1 above)
Comment (Adamantios) [reply packages/valory/services/apy_estimation_demo/service.yaml:114]: Why add them?

File: packages/valory/services/apy_estimation/service.yaml [hunk 4]
```diff
@@ -42,8 +40,8 @@ type: skill
         n_observations: ${SERVICE_APY_N_OBSERVATIONS:int:120}
         history_end: null
         ipfs_domain_name: /dns/178.62.4.138/tcp/5001/http
-        service_registry_address: ${SERVICE_REGISTRY:str:null}
-        share_tm_config_on_startup: ${USE_ACN:bool:false}
+        service_registry_address: ${SERVICE_REGISTRY_ADDRESS:str:null}
+        share_tm_config_on_startup: ${SHARE_TM_CONFIG_ON_STARTUP:bool:false}
```
Comment (Adamantios) [inline packages/valory/services/apy_estimation/service.yaml:44]: Is this a requirement of the framework?

**Review by Adamantios — APPROVED** [review]

**Review by DavidMinarsch — APPROVED** [review]

**Pull request merged**